File: manila/src/main/java/cz/fb/manaus/manila/ManilaBet.java
package cz.fb.manaus.manila;

import org.springframework.beans.factory.annotation.Qualifier;

import java.lang.annotation.ElementType;
import java.lang.annotation.Retention;
import java.lang.annotation.RetentionPolicy;
import java.lang.annotation.Target;

@Qualifier
@Target({ElementType.TYPE, ElementType.FIELD, ElementType.CONSTRUCTOR})
@Retention(RetentionPolicy.RUNTIME)
public @interface ManilaBet {
}

File: manila/src/main/java/cz/fb/manaus/manila/ManilaBet.kt
package cz.fb.manaus.manila

import org.springframework.beans.factory.annotation.Qualifier

@Qualifier
@Target(
        AnnotationTarget.CLASS,
        AnnotationTarget.FILE,
        AnnotationTarget.FUNCTION,
        AnnotationTarget.VALUE_PARAMETER,
        AnnotationTarget.FIELD,
        AnnotationTarget.CONSTRUCTOR)
@Retention
annotation class ManilaBet
